mlagents/envs/socket_communicator.py

```diff
@@ -3,12 +3,12 @@
 import struct
 
 from .communicator import Communicator
-from communicator_objects import UnityMessage, UnityOutput, UnityInput
+from .communicator_objects import UnityMessage, UnityOutput, UnityInput
 from .exception import UnityTimeOutException
 
 
 logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger("unityagents")
+logger = logging.getLogger("mlagents.envs")
```

Comment:
Not sure what the logger should be, but `mlagents.envs` is too specific. We can spell out `ML-Agents` unless we want a different logger for each package (I am also okay with that)

Reply:
I was thinking we could have a different logger for `mlagents.envs` and `mlagents.trainers` which are the two main packages we ship.